### ISSUE TITLE
Idle document crashes CODE

### ISSUE THREAD

**Issue by rpear** [CollaboraOnline/online#3550]
**Describe the bug**
If the computer is left alone for a period until the "document is idle, click to continue" pop up appears, CODE does not recover. No more tiles appear, scroll up and down and then all text disappears. The side bar no longer toggles on or off. However the menu buttons still appear. It is still possible to scroll up and down through the pages, but the only text that appears is the new big page numbers in the middle of each page.

**Desktop / browser / version**
Chromebook, Chrome, Version 94.0.4606.104 (Official Build) (64-bit)

**CODE version**
LOOLWSD 21.11.0.0snapshot (git hash: 570c31d) Served by: 5ebf03b4
LOKit built on a great technology base Collabora Office 21.06.5.2 (git hash: d1a0bb1) "Ubuntu 18.04.6 LTS"

**Comment by Ezinnem** [comment]
I had a different experience.

I think the server disconnects during the idle time.

When I click to continue after the idle time, I get a "connecting" dialog which stays for a long while.

**Comment by joergmschulz** [comment]
same for 21.11.0.1
<img width="643" alt="image" src="https://user-images.githubusercontent.com/6523646/142460722-89f9ba57-f21e-409d-897f-c399f6659abb.png">

**Comment by joergmschulz** [comment]
it seems 
COOLWSD version:
```
21.11.0.1 (git hash:f815ff45)
LOKit version:
Collabora Office 21.06.6.1 (git hash:8037a24)
Ubuntu 18.04.6 LTS
```

fix this issue (probably https://github.com/CollaboraOnline/online/pull/3683 fixed it???)

**Comment by thebearon** [comment]
This should be fixed by #3692, RC2 with the fix will be out shortly.
The idle timeout can be configured in the config file, look for a `3600` (the default 1 hour in seconds), and change it to something smaller if you wish to test it.
